packages/embed/src/utils/get-share-analytics-instance.ts

```diff
@@ -0,0 +1,28 @@
+const GA_TYPE_MESSAGE = 'ga-client-id'
+
+export const setupGoogleAnalyticsInstanceSharingFeature = (iframe: HTMLIFrameElement, embedId: string) => {
+  const sendGaIdMessage = (gaClientId: string) => {
```

Comment:
I would move this function outside of `setupGoogleAnalyticsInstanceSharingFeature` and add `embedId` as param.